src/h5web/visualizations/line/hooks.ts

```diff
@@ -4,15 +4,29 @@ import { useThree } from 'react-three-fiber';
 import { Domain } from '../shared/models';
 import { findDomain, getAxisScale } from '../shared/utils';
 import { AxisSystemContext } from '../shared/AxisSystemProvider';
-import { useVisProps } from '../../dataset-visualizer/VisProvider';
+import {
+  useVisProps,
+  useFlatValues,
+} from '../../dataset-visualizer/VisProvider';
 
 export function useData(): number[] {
-  const { rawValues, rawDims } = useVisProps();
+  const { rawValues, rawDims, mapping } = useVisProps();
+  const values = useFlatValues();
 
   if (rawDims.length === 1) {
     return rawValues;
   }
 
+  if (rawDims.length === 2) {
+    if (mapping.x === 1) {
+      return rawValues[mapping.slicingIndices[0]];
+    }
```

Comment:
If the dataDimension is the deepest one, we don't have to think really hard on how to access the data: it is only `data[slicingIndexForDim0]`